samples/glean/src/main/res/values/strings.xml

```diff
@@ -15,9 +15,6 @@
         to attach a dictionary of values to the extras of the event ping. See MainActivity for
         where all of this is happening.
     </string>
-    <string name="send_ping_info">
```

Comment:
Can we also remove the `send_baseline_ping` string?

Reply:
Indeed.  Thanks.